internal/kafka/avail_status_msg.go

```diff
@@ -40,10 +41,10 @@ func genericMessage(ctx context.Context, m any, key string, topic string) (Gener
 		Value: payload,
 		// Keep headers written in lowercase to match sources comparison.
 		Headers: GenericHeaders(
-			"content-type", "application/json",
-			"x-rh-identity", identity.IdentityHeader(ctx),
-			"x-rh-sources-org-id", id.Identity.OrgID,
-			"x-rh-sources-account-number", id.Identity.AccountNumber,
+			"Content-Type", "application/json",
+			"X-RH-Identity", identity.IdentityHeader(ctx),
```

Comment:
By the way, the X-RH-Identity is encoded in some way? I can see that we are decoding it with `base64.StdEncoding.DecodeString` somewhere else.

Reply:
Check documentation of the IdentityHeader function, yes, this is a base64-encoded JSON.